.github/workflows/linux_unit_tests.yaml

```diff
@@ -10,7 +10,67 @@ on:
       - develop
       - releases/**
 jobs:
+  # Validate that the code can be run on all the Python versions
+  # supported by Spack
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Install Python Packages
+      run: |
+        pip install --upgrade pip
+        pip install --upgrade vermin
+    - name: Minimum Version (Spack's Core)
+      run: vermin --backport argparse --backport typing -t=2.6- -t=3.5- -v lib/spack/spack/ lib/spack/llnl/ bin/
```

Comment:
vermin cli doesn't have those options, but I wish it did. The vermin developer knows about Spack and contributes new versions when he releases them, so it may be easier to contribute directly to vermin instead of writing a wrapper script.